packages/server/email/teamLimitsEmailCreator.tsx

```diff
@@ -0,0 +1,113 @@
+import LockedEmail from 'parabol-client/modules/email/components/LimitsEmails/LockedEmail'
+import SevenDayWarningEmail from 'parabol-client/modules/email/components/LimitsEmails/SevenDayWarningEmail'
+import ThirtyDayWarningEmail from 'parabol-client/modules/email/components/LimitsEmails/ThirtyDayWarningEmail'
+import {PALETTE} from 'parabol-client/styles/paletteV3'
+import React from 'react'
+import ReactDOMServer from 'react-dom/server'
+import makeAppURL from '../../client/utils/makeAppURL'
+import appOrigin from '../appOrigin'
+import {TeamLimitsEmailType} from '../billing/helpers/sendTeamsLimitEmail'
+import {analytics} from '../utils/analytics/analytics'
+import emailTemplate from './emailTemplate'
+
+const textOnlySummary = (props: Props) => {
+  const {preferredName, orgId, emailType, orgName, stickyTeamCount} = props
+  const billingURL = makeAppURL(appOrigin, `/me/organizations/${orgId}/billing`, {
+    searchParams: {
+      utm_source: 'notification email',
+      utm_medium: 'email',
+      utm_campaign: 'locked-email'
+    }
+  })
+
+  const emailTextLookup = {
+    thirtyDayWarning: `Hi ${preferredName} 👋
+
+    This is a friendly note to let you know that ${orgName} has officially reached ${stickyTeamCount} active teams on Parabol - congrats! We love to see organizations finding value in Parabol and improving their teams in the process.
```

Comment:
I think we don't need to use real-time `stickyTeamCount` value, as everywhere else we are talking about 2 teams limit. I think it is ok to just say that they are over 2 teams, without specifying exact number. I would use `MAX_PERSONAL_TIER_TEAMS`

 What do you think?